db/re/item_db.txt

```diff
@@ -12611,26 +12617,151 @@
+31580,C_Autumn_Flavor,C Autumn Flavor,4,0,,0,,0,,0,0xFFFFFFFF,63,2,2048,,1,0,1792,{},{},{}
+31581,C_Jack_Upper,C Jack Upper,4,0,,,,0,,0,0xFFFFFFFF,63,2,1024,,1,0,1415,{},{},{}
+31582,C_Jirant_Circlet,C Jirant Circlet,4,0,,,,0,,0,0xFFFFFFFF,63,2,1024,,1,0,1644,{},{},{}
+31583,C_Will_O_Wisp_Lower,C Will O Wisp Lower,4,0,,,,0,,0,0xFFFFFFFF,63,2,4096,,1,0,880,{},{ sc_start SC_STRANGELIGHTS,INFINITE_TICK,0; },{}
+31586,C_Poporing_Muffler,C Poporing Muffler,4,0,,,,0,,0,0xFFFFFFFF,63,2,4096,,1,0,1797,{},{},{}
+31588,C_Snow_Flower,C Snow Flower,4,0,,,,0,,0,0xFFFFFFFF,63,2,1024,,1,0,1799,{},{},{}
+31589,C_Golden_Majestic_Goat,C Golden Majestic Goat,4,0,,0,,0,,0,0xFFFFFFFF,63,2,1024,,1,0,1800,{},{},{}
+31590,C_Designer_Brush,C Designer's Brush,4,0,,,,0,,0,0xFFFFFFFF,63,2,4096,,1,0,1291,{},{},{}
+31593,C_Yellow_Scarf,C Yellow Scarf,4,0,,,,0,,0,0xFFFFFFFF,63,2,4096,,1,0,1170,{},{},{}
+31594,C_Red_Pencil_In_Mouth,C Red Pencil in Mouth,4,0,,,,0,,0,0xFFFFFFFF,63,2,4096,,1,0,931,{},{},{}
+31595,C_Book_File_Hat,C Book File Hat,4,0,,,,0,,0,0xFFFFFFFF,63,2,1024,,1,0,423,{},{},{}
+31596,C_Spider_Seduction,C Spider Seduction,4,0,,,,0,,0,0xFFFFFFFF,63,2,1024,,1,0,1711,{},{},{}
+31597,C_Golden_Fish_Hat,C Golden Fish Hat,4,0,,,,0,,0,0xFFFFFFFF,63,2,1024,,1,0,1645,{},{},{}
+31598,C_Forest_Guide,C Forest Guide,4,0,,,,0,,0,0xFFFFFFFF,63,2,1024,,1,0,1802,{},{},{}
+31599,C_Medium_Wave,C Medium Wave,4,0,,,,0,,0,0xFFFFFFFF,63,2,4096,,1,0,1803,{},{},{}
+31600,C_Kishu_Dog,C Kishu Dog,4,0,,,,0,,0,0xFFFFFFFF,63,2,2048,,1,0,1804,{},{},{}
+31601,C_Under_Rim_Glasses_Blue,C Under Rim Glasses Blue,,0,,,,0,,0,0xFFFFFFFF,63,2,2048,,1,,1805,{},{},{}
+31602,C_Midgarts_Glory,C Rune Midgarts Glory,,0,,,,0,,0,0xFFFFFFFF,63,2,1024,,1,,1807,{},{},{}
+31606,C_Autumn_Headband,C Autumn Headband,4,0,,0,,0,,0,0xFFFFFFFF,63,2,1024,,1,0,1119,{},{},{}
+31607,C_Cat_Ear_Witch_Hat,C Cat Ear Witch Hat,4,0,,,,0,,0,0xFFFFFFFF,63,2,1024,,1,0,1808,{},{},{}
+31608,C_Magicstone_of_Grace,C Magic Stone of Grace,4,0,,,,0,,0,0xFFFFFFFF,63,2,7168,,1,0,1420,{},{},{}
+31609,C_Noble_Mask,C Noble Mask,,0,,,,0,,0,0xFFFFFFFF,63,2,2048,,1,,1409,{},{},{}
+31610,C_Yellow_Wizardry_Hat,C Yellow Wizardry Hat,4,0,,,,0,,0,0xFFFFFFFF,63,2,1024,,1,0,286,{},{},{}
+31611,C_Snake_Lord_Stole,C Snake Lord Stole,4,0,,,,0,,0,0xFFFFFFFF,63,2,4096,,1,0,1810,{},{},{}
+31612,C_Large_Sorcerer_Crown,C Large Sorcerer Crown,4,0,,,,0,,0,0xFFFFFFFF,63,2,1024,,1,0,1811,{},{},{}
 //
 32005,Illusion_Pole_Axe,Illusion Pole Axe,5,0,,1500,190,,3,2,0x00004082,63,2,2,4,120,1,4,{ .@r = getrefine(); bonus bStr,3; bonus bInt,2; bonus bDex,2; if (.@r >= 13) { .@val = -2000; } else if (.@r >= 9) { .@val = -1000; } bonus2 bSkillCooldown,"RK_HUNDREDSPEAR",.@val; },{},{}
+32013,Metal_Stick,Metal Stick,5,0,0,2000,180,,3,3,0x00004000,63,2,2,4,150,1,4,{ .@r = getrefine(); bonus2 bSkillAtk,"LG_CANNONSPEAR",10; bonus2 bSkillAtk,"LG_OVERBRAND",(.@r > 8 ? 15 : (.@r > 6 ? 10 : 0)); if(.@r >= 12){ bonus bDelayrate,-15; } },{},{}
```

Comment:
- The damage of `LG_OVERBRAND` should be increased by 25% (10+15%) at +9 or above.
- `Jobs: 3rd classes swordsman` means the equippable jobs should be `0x00004080` and the equippable upper-types should be `56`.
```
	Knight         (2^07): 0x00000080
	Crusader       (2^14): 0x00004000
```
```
	 8: Third classes (no Transcedent-Third or Third-Baby classes)
	16: Transcedent-Third classes
	32: Third-Baby classes
```